### ISSUE TITLE
HTTP/3: Remove try/catch around IsSupported property

### ISSUE THREAD

**Issue by JamesNK** [dotnet/aspnetcore#41042]
Steve added try/catch here https://github.com/dotnet/aspnetcore/pull/40376

IsSupported no longer throws: https://github.com/dotnet/runtime/pull/67036

Try/catch can be removed.

**Comment by JamesNK** [comment]
User reported this is broken again: https://github.com/dotnet/aspnetcore/pull/41123#issuecomment-1133983612

@SteveSandersonMS Could you double-check whether this is working or not?

**Comment by SteveSandersonMS** [comment]
@JamesNK Yes, it's working - see https://github.com/dotnet/aspnetcore/pull/41123#issuecomment-1135884829
